openmdao/utils/hooks.py

```diff
@@ -113,7 +175,7 @@ def add_hooks(self, hooks):
         self.pre_hooks = sorted(self.pre_hooks, key=lambda x: x._stamp)
         self.post_hooks = sorted(self.post_hooks, key=lambda x: x._stamp)
 
-    def _run_hooks(self, hooks):
+    def _run_hooks(self, hooks, args, kwargs, ret=None):
```

Comment:
should update the docstring for the new args...

also should probably add a docstring to `_HookMeta`

Reply:
done